doc/interop-test-descriptions.md

```diff
@@ -1011,16 +1011,49 @@ languages. Therefore they are not part of our interop matrix.
 #### rpc_soak
 
 The client performs many large_unary RPCs in sequence over the same channel.
```

Comment:
We should actually define the expected test behavior, like done in the other tests, instead of just implying it from the arguments.

Reply:
done